src/core/decorators/cacheToFile.ts

```diff
@@ -64,16 +64,6 @@ export function CacheToFile({ fileKey }: { fileKey: string }) {
         throw new Error('Metadata file not found')
       }
 
-      logger.debug(
```

Comment:
This made the logs impossible to read, and the more files we add the worse it'll get. If file could not be loaded it would error anyway.